Apps/Src/Voltage.c

```diff
@@ -98,15 +99,30 @@ SafetyStatus Voltage_CheckStatus(void){
  * @return pointer to index of modules that are in danger
  */
 SafetyStatus *Voltage_GetModulesInDanger(void){
-	static SafetyStatus checks[NUM_BATTERY_MODULES];
-	uint32_t open_wires = Voltage_GetOpenWire();
+	static SafetyStatus checks[TOTAL_PINS];
+	Voltage_GetOpenWire();
+	uint32_t wires;
+	uint32_t openWireIdx = 0;
+	//put all the bits from each minion's system_open_wire variable into one variable
+	for(int k = 0; k < NUM_MINIONS; k++){
+		wires = (Minions[k].system_open_wire & 0x1FF);	//there are at most 8 modules per IC, bit 0 is GND
+		for(int l = 0; l < NUM_PINS_PER_IC; l++){
```

Comment:
Can you use a different variable name than `l`? It looks a lot like the number `1` in some fonts